functions/venue.js

```diff
@@ -300,6 +300,26 @@ exports.setAuditoriumSections = functions.https.onCall(
     // Removing sections if needed
     const numberOfSectionsToRemove = -1 * numberOfSectionsToAdd;
     for (let i = 1; i <= numberOfSectionsToRemove; i++) {
+      // to remove subcollection seatedSectionUsers we need to remove all its objects
+      const batchForSections = admin.firestore().batch();
+      const sectionId = sections[currentNumberOfSections - i].id;
+      admin
+        .firestore()
+        .collection("venues")
+        .doc(venueId)
+        .collection("sections")
+        .doc(sectionId)
+        .collection("seatedSectionUsers")
+        .listDocuments()
+        .then((array) => {
```

Comment:
How will this entire loop act if a section with 200 seated users is being deleted?
Will it hit a limit of time or memory?
Will an error in this prevent adding or deleting sections?